src/modules/iKinGazeCtrl/src/utils.cpp

```diff
@@ -360,6 +363,52 @@ Vector ExchangeData::get_imu()
     return _imu;
 }
 
+/************************************************************************/
+std::pair<Vector,bool>  ExchangeData::get_gyro() {
+    std::pair<Vector, bool> ret;
+    Vector gyro(3,0.0);
+    if (useMASClient) {
+
+        if (! iGyro) {
+            ret.second = false;
+        }
+        else {
+            double ts;
+            ret.second =  iGyro->getThreeAxisGyroscopeStatus(0) == MAS_OK;
+            ret.second &= iGyro->getThreeAxisGyroscopeMeasure(0, gyro, ts);
```

Comment:
```suggestion
            ret.second = iGyro->getThreeAxisGyroscopeMeasure(0, gyro, ts);
```

`ret.second` is undefined before this operation, as far as I see.

Reply:
It is valued with the status of the sensor just the line before, I don't know if it is the case to check the status of the sensor every time (@traversaro).

Reply:
I was blind 👀